.cirrus.yml

```diff
@@ -134,59 +145,79 @@ plugin_qa_task:
     - mvn verify -Drevision=${PROJECT_VERSION} -Dsonar.runtimeVersion=${SQ_VERSION} -Dmaven.test.redirectTestOutputToFile=false -B -e -V
   cleanup_before_cache_script: cleanup_maven_repository
 
-qa_os_win_task:
-  <<: *WINDOWS_VM_DEFINITION
+ruling_task:
+  name: Integration Test (Ruling)
   <<: *ONLY_SONARSOURCE_QA
-  depends_on:
-    - build
-  maven_cache:
-    #windows cache is buggy if using ${CIRRUS_WORKING_DIR}
-    folder: ~/.m2/repository
-  build_script:
-    - source cirrus-env CI
-    - mvn.cmd clean verify
+  <<: *DEPENDS_ON_BUILD
+  <<: *QA_CONTAINER_DEFINITION
+  <<: *MAVEN_CACHE
+  env:
+    CIRRUS_CLONE_DEPTH: 1
+    matrix:
+      - PHP_PROJECT: "Flysystem"
+      - PHP_PROJECT: "Monica"
+      - PHP_PROJECT: "PhpCodeSniffer"
+      - PHP_PROJECT: "PhpMailer"
+      - PHP_PROJECT: "Psysh"
+      - PHP_PROJECT: "PhpWord"
+      - PHP_PROJECT: "RubixML"
+  submodules_script:
+    - git submodule update --init
+  ruling_script:
+    - source cirrus-env QA
+    - source set_maven_build_version $BUILD_NUMBER
+    - cd its/ruling
+    - mvn test -Dtest=PhpGeneralRulingTest#test${PHP_PROJECT} -Drevision=${PROJECT_VERSION} -Dsonar.runtimeVersion=LATEST_RELEASE -Dmaven.test.redirectTestOutputToFile=false -B -e -V
   cleanup_before_cache_script: cleanup_maven_repository
 
-ruling_task:
-  depends_on:
-    - build
+pr_analysis_task:
+  name: Integration Test (PR Analysis)
   <<: *ONLY_SONARSOURCE_QA
-  eks_container:
-    <<: *QA_CONTAINER_DEFINITION
-    cpu: 4
-    memory: 8G
+  <<: *DEPENDS_ON_BUILD
+  <<: *QA_CONTAINER_DEFINITION
+  <<: *MAVEN_CACHE
   env:
     CIRRUS_CLONE_DEPTH: 1
-  maven_cache:
-    folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   submodules_script:
     - git submodule update --init
   ruling_script:
     - source cirrus-env QA
     - source set_maven_build_version $BUILD_NUMBER
     - cd its/ruling
-    - mvn verify -Drevision=${PROJECT_VERSION} -Dsonar.runtimeVersion=LATEST_RELEASE -Dmaven.test.redirectTestOutputToFile=false -B -e -V
+    - mvn test -Dtest=PhpPrAnalysisTest -Drevision=${PROJECT_VERSION} -Dsonar.runtimeVersion=LATEST_RELEASE -Dmaven.test.redirectTestOutputToFile=false -B -e -V
   cleanup_before_cache_script: cleanup_maven_repository
+
+mend_scan_task:
+  name: Mend SCA
+  <<: *ONLY_SONARSOURCE_QA
+  <<: *DEPENDS_ON_BUILD
+  <<: *CONTAINER_DEFINITION
+  <<: *MAVEN_CACHE
+  env:
+    <<: *MEND_SCAN_SECRETS
+  whitesource_script:
+    - source cirrus-env QA
+    - source set_maven_build_version $BUILD_NUMBER
+    - mvn clean install -DskipTests
+    - source ws_scan.sh
+  allow_failures: "true"
+  always:
+    ws_artifacts:
+      path: "whitesource/**/*"
+
  
 promote_task:
+  name: Promote Artifact
   depends_on:
-    - ruling
-    - plugin_qa
-    - qa_os_win
-    - ws_scan
+    - Integration Test (Ruling)
+    - Integration Test (PR Analysis)
+    - Integration Test (Plugin)
```

Comment:
IMO the `Mend SCA` is missing here as dependency

Reply:
Due to the SCA is not a mandatory step to be successful I would like to remove it from the 'depend on' list.